### PR TITLE
Daemonizing

### PR DESCRIPTION
**Pull request by ChristopherMacGown**
This branch adds daemonization to roundabout, it can be used with:

bin/roundabout start|stop|run|restart

It also forces configs to be in json only, configs can and must be updated with:

bin/roundabout update_config --config_file=/path/to/config_file

### CODE REVIEW

File: roundabout/__init__.py [hunk 1]
```diff
@@ -2,73 +2,102 @@
...
-                    except git_client.GitException, e:
-                        pull_request.close(git_client.MERGE_FAIL_MSG % e)
+        pull_requests = github.pull_requests
+        pull_requests = [(u, p) for u, p
+                                in pull_requests.items()
+                                if p.lgtm(github.approvers)]
+
+        if not pull_requests:
+            log.info("No work to do, sleeping.")
+            time.sleep(30)
```
Comment (larsbutler) [inline roundabout/__init__.py:62]: If there's no work to do, shouldn't the rest of the loop be explicitly skipped (with a 'continue')?

**Comment by ChristopherMacGown** [comment]
lgtm

**Comment by ChristopherMacGown** [comment]
Build failed, rejecting.

http://hudson.atomicpony.com/job/roundabout/112/

**Comment by ChristopherMacGown** [comment]
Merge failed due to out-of-date .cfg

**Comment by ChristopherMacGown** [comment]
lgtm

**Comment by ChristopherMacGown** [comment]
Build failed, rejecting.

http://hudson.atomicpony.com/job/roundabout/114/

**Comment by ChristopherMacGown** [comment]
lgtm

**Comment by ChristopherMacGown** [comment]
Build successful! Merged!

**Pull request merged**